init.sqf

```diff
@@ -114,12 +114,10 @@ f_script_briefing = [] execVM "briefing.sqf";
 // ====================================================================================
 
 // F3 - Automatic Body Removal
-// Credits: Please see the F3 online manual (http://www.ferstaberinde.com/f3/en/)
 
-f_var_removeBodyDelay = 180;
-f_var_removeBodyDistance = 500;
-f_var_doNotRemoveBodies = [];
-[] execVM "f\removeBody\f_addRemoveBodyEH.sqf";
+//Exclude units from automatic body/wreck removal:
+removeFromRemainsCollector playableUnits;

```

Comment:
Note: Effects of `removeFromRemainsCollector` are global. 
While this means that this gets called for every player which is not ideal (because it should only have to happen once (on the server)), 
it is also good _because_ it gets called for JIP players (especially useful for slots that are not marked as AI) so that they are also excluded from the remains collector)